ts/screens/wallet/payment/TransactionSummaryScreen.tsx

```diff
@@ -0,0 +1,170 @@
+import Markdown from "../../../components/ui/Markdown";
+import PaymentSummaryComponent from "../../../components/wallet/PaymentSummaryComponent";
+import I18n from "../../../i18n";
+import { Dispatch } from "../../../store/actions/types";
+import { confirmSummary } from "../../../store/actions/wallet/payment";
+import { GlobalState } from "../../../store/reducers/types";
+import {
+  currentAmountSelector,
+  getInitialAmount,
+  getRptId,
+  paymentReasonSelector,
+  paymentRecipientSelector
+} from "../../../store/reducers/wallet/payment";
+import variables from "../../../theme/variables";
+import { UNKNOWN_RECIPIENT, UNKNOWN_RPTID } from "../../../types/unknown";
+
+type ReduxMappedStateProps = Readonly<{
+  paymentRecipient: EnteBeneficiario;
+  paymentReason: string;
+  rptId: RptId;
+  originalAmount: AmountInEuroCents;
+  updatedAmount: AmountInEuroCents;
+}>;
+
+type ReduxMappedDispatchProps = Readonly<{
+  confirmSummary: () => void;
+}>;
+
+type OwnProps = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+type Props = OwnProps & ReduxMappedStateProps & ReduxMappedDispatchProps;
+
+const formatMdRecipient = (e: EnteBeneficiario): React.ReactNode => (
+  <Markdown>
+    {`
+  **${I18n.t("wallet.firstTransactionSummary.entity")}**\n
+  ${e.denominazioneBeneficiario} - ${e.denomUnitOperBeneficiario}\n
+  ${e.indirizzoBeneficiario} n. ${e.civicoBeneficiario}\n
+  ${e.capBeneficiario} ${e.localitaBeneficiario} (${e.provinciaBeneficiario})
+  `}
+  </Markdown>
+);
+
+const formatMdPaymentReason = (p: string): React.ReactNode => (
+  <Markdown>
+    {`
+  **${I18n.t("wallet.firstTransactionSummary.object")}**\n
+  ${p}
+  `}
+  </Markdown>
+);
+
+const formatMdInfoRpt = (r: RptId): React.ReactNode => (
+  <Markdown>
+    {`
+  **IUV:** ${PaymentNoticeNumberFromString.encode(r.paymentNoticeNumber)}\n
+  **Recipient Fiscal Code:** ${r.organizationFiscalCode}`}
+  </Markdown>
+);
+
+class TransactionSummaryScreen extends React.Component<Props, never> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  private goBack() {
+    this.props.navigation.goBack();
+  }
+
+  public render(): React.ReactNode {
+    return (
+      <Container>
+        <AppHeader>
+          <Left>
+            <Button transparent={true} onPress={() => this.goBack()}>
+              <IconFont name="io-back" size={variables.iconSize3} />
+            </Button>
+          </Left>
+          <Body>
+            <Text>{I18n.t("wallet.firstTransactionSummary.header")}</Text>
+          </Body>
+        </AppHeader>
+
+        <Content noPadded={true}>
+          <PaymentSummaryComponent
+            navigation={this.props.navigation}
+            amount={`${AmountInEuroCentsFromNumber.encode(
+              this.props.originalAmount
+            )}`}
+            updatedAmount={`${AmountInEuroCentsFromNumber.encode(
+              this.props.updatedAmount
+            )}`}
+          />
+          {formatMdRecipient(this.props.paymentRecipient)}
```

Comment:
suggestion: have `formatMdRecipient` and the others return a markdown string, keep the `<markdown>` tag here in the main TSX